explainer-server/app/controllers/ApiController.scala

```diff
@@ -41,16 +41,23 @@ class ExplainerApiImpl(
   val explainerDB = new ExplainerDB(config)
   val explainerStore = new ExplainerStore(config)
 
-  def publishExplainerToKinesis(explainer: Atom, actionMessage: String, atomPublisher: AtomPublisher) = {
+  def publishExplainerToKinesis(explainer: Atom, actionMessage: String, atomPublisher: AtomPublisher, eventType: EventType = EventType.Update) = {
     if (config.publishToKinesis) {
       val event = ContentAtomEvent(explainer, EventType.Update, DateTime.now.getMillis)
       atomPublisher.publishAtomEvent(event) match {
-        case Success(_) => Logger.info(s"$actionMessage succeeded")
-        case Failure(err) => Logger.error(s"$actionMessage failed", err)
+        case Success(_) => {
+          Logger.info(s"$actionMessage succeeded")
+          "success"
```

Comment:
@LATaylor-guardian @philmcmahon 

Does either of you mind

```
sealed trait PublishResult
case object PRSuccess extends PublishResult
case object PRFail extends PublishResult
case object PRDisabled extends PublishResult
```

?

(because `Success` conflicts with `scala.util.Success`)